napalm_logs/base.py

```diff
@@ -40,14 +40,6 @@
 
 class NapalmLogs:
     def __init__(self,
-                 address='0.0.0.0',
```

Comment:
No, please don't remove these key field and introduce another hierarchy let's keep things linear and simple. There's no point in removing two fields and replacing it with a dictionary that introduces unnecessary nesting levels.